test/functorch/test_eager_transforms.py

```diff
@@ -307,6 +307,7 @@ def foo(x):
         with self.assertRaisesRegex(RuntimeError, 'mutate a captured Tensor'):
             grad(foo)(x)
 
+    @skipIfTorchDynamo("Debugging")
```

Comment:
This is double grad issue, debugging now.

Reply:
Wasn't the plan for this change to be semantics preserving?

Reply:
No, because we updated the ```is_allowed``` generating rule, though I think it doesn't matter. For this particular issue, I suspect it comes from the refactor in ```torch.py```, I have to figure out the root cause at least.

Reply:
Resolved.